tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -56,7 +58,16 @@ def regex_match(regex, string):
     return re.match(regex, string)
 
 
```

Comment:
```suggestion
# Fixtures
```

Reply:
This comment follows the file standard, however, maybe they are not very useful. It should be necessary to standardize all tests structure and commentaries with the rest of the team in the future.